src/lib/rest/curl.cpp

```diff
@@ -0,0 +1,335 @@
+/*
```

Comment:
Maybe better name names:

src/lib/rest/curlSem.cpp

as plain "curl" is too broad.

Reply:
...and similar comment for curl.h -> curlSem.h

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3869/commits/5f663bcfa195d8da60f4c74e939e7d8c19b4bb9d